src/github/credentials.ts

```diff
@@ -76,24 +52,16 @@ export class CredentialStore implements vscode.Disposable {
 			return true;
 		}
 
-		const server = new GitHubServer(host);
-		const token = await getToken(host);
-		let octokit: GitHub | undefined = undefined;
+		const existingSessions = await vscode.authentication.getSessions(AUTH_PROVIDER_ID, SCOPES);
 
-		if (token) {
-			if (await server.validate(token)) {
-				octokit = await this.createHub({ host, token });
-			} else {
-				Logger.debug(`Token is no longer valid for host ${host}.`, 'Authentication');
-			}
+		if (existingSessions.length) {
+			const token = await existingSessions[0].getAccessToken();
```

Comment:
Why is it always the session at 0?

Reply:
This was under the assumption that all sessions from the github auth provider would be for the same account. Signing into multiple accounts with the provider is challenging, because github only allows you to be signed into one account at at time in the browser. so I guess theoretically you could sign in, then logout in the browser and trigger sign in again from vscode

perhaps I should show a quickpick if there are different accounts available